mscore/musescore.cpp

```diff
@@ -4424,6 +4439,10 @@ void MuseScore::readSettings()
 
       MuseScore::restoreGeometry(this);
 
+      // grab the visible state before the beginGroup
+      // this awkwardness to ensure state saved before code changes should
```

Comment:
Could you please add a brief description of which code changes are mentioned in this comment? It will be possible to find this out via `git blame` but it will be even better if one could understand this comment without searching through the changes history.